images/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# update ca certificates in case they are injected with a volume mount
+function updateCerts() {
```

Comment:
No I think that is a great point. The only reason I can think of is because the entrypoint is really tied to the docker image where update certs make sense if you mount them as volumes.
The entrypoint can also provide a place where we can include extra behaviour only for containers. For example, a nice addition here might be to export env registering signal handling on `run.sh` by default, since we know we are going to need it. That can eliminate the code on ARC controller keeping track of those envs.

I don't hold a strong opinion. Right now, the `entrypoint.sh` is just a tiny wrapper.